modules/ROOT/pages/troubleshoot-api-provisioning-issues.adoc

```diff
@@ -52,3 +56,45 @@ To resolve this issue:
 . Delete the API.
 . Resolve the username and password error.
 . Re-create the API.  
+
+
+[[provision-call-failed]]
+== Provision call failed
+
+When you create an API, the Asset(in exchange) and/or API(in API Manager) are not successfuly created on the Anypoint Platform
```

Comment:
```suggestion
When you create an API, the Asset (in exchange) and/or API (in API Manager) are not successfully created on the Anypoint Platform
```

Also, do you mind to clarify when we are talking about API provision and API instances?

```suggestion
When you create an API provision, the Asset (in exchange) and/or API instance (in API Manager) are not successfully created on the Anypoint Platform
```